File: backend-SpringBoot/src/main/java/com/example/employee_management/Models/Employee.java
package com.example.employee_management.Models;

import jakarta.persistence.*;
import jakarta.validation.constraints.*;
import lombok.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Employee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @NotBlank(message = "First name is required!")
    @Column(nullable = false)
    private String firstName;

    @NotBlank(message = "Last name is required!")
    @Column(nullable = false)
    private String lastName;

    @NotBlank(message = "Phone number is required!")
    @Column(nullable = false)
    private String phone;

    @NotBlank(message = "E-mail is required!")
    @Email(message = "Invalid e-mail address!")
    @Column(nullable = false, unique = true)
    private String email;

    @NotBlank(message = "Position is required!")
    @Column(nullable = false)
    private String position;

    @Version
    @Column(nullable = false)
    private Long version = 0L;
}

File: backend-C#/EmployeeManagement/Models/Employee.cs
using System.ComponentModel.DataAnnotations;

namespace EmployeeManagement.Models
{
    public class Employee
    {
        public int Id { set; get; }

        [Required(ErrorMessage = "First name is required!")]
        // [Length(0,50)]
        public string FirstName { set; get; }
        
        [Required(ErrorMessage = "Last name is required!")]
        public string LastName { set; get; }
        
        [Required(ErrorMessage = "Phone number is required!")]
        // [Range(0, 100000)]
        public string Phone {  set; get; }
        
        [Required(ErrorMessage = "E-mail number is required!")]
        [EmailAddress(ErrorMessage = "Invalid e-mail address!")]
        public string Email { set; get; }
        
        [Required(ErrorMessage = "Position is required!")]
        public string Position { set; get; }

    }
}
